tests/fetch-browser.spec.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+
+var fetch = require('../build/fetch-browser')();
+
+var good = 'hello world. 你好世界。\n';
+
+function responseToText(response) {
+  if (response.status >= 400) throw new Error('Bad server response');
+  return response.text();
+}
+
+describe('fetch', function() {
+
+  it('should make requests', function (done) {
+    fetch('https://s3.amazonaws.com/bodylabs-edge/unittest/fetch/hello.1.0.0.txt')
```

Comment:
Rather than rely on s3 to host this, I'd prefer the test server to host the file, or better yet, [use sinon](http://sinonjs.org/docs/#server) to hijack the requests. If you're stuck for time, I don't mind following this PR up with that. Your call.